src/Devices/Artemis.Plugins.Devices.Wooting/Services/WootingAnalogDevice.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using WootingAnalogSDKNET;
+using RGB.NET.Core;
+
+namespace Artemis.Plugins.Devices.Wooting;
+
+public class WootingAnalogDevice
+{
+    public Dictionary<LedId, float> AnalogValues { get; }
+    public DeviceInfo Info { get; }
+
+    public WootingAnalogDevice(DeviceInfo info)
+    {
+        Info = info;
+        AnalogValues = new();
```

Comment:
```suggestion
        AnalogValues = new Dictionary<LedId, float>();
```
Better to only use the short-hand `new` if the type is evident, which here it is not